src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,119 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+
+    private static int MAX_ARRAY_SIZE = 4;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
class fields have default value, int default value is 0